test/web/10-api.js

```diff
@@ -698,6 +698,91 @@ describe('Profile Manager API', () => {
       error.name.should.equal('TypeError');
       error.message.should.contain('profileId');
     });
+    it('should fail if no edvId', async () => {
+      const hmac = {
```

Comment:
In this section of code: https://github.com/digitalbazaar/bedrock-web-profile-manager/pull/61/files#diff-1307d27cbc90d4f1584d2e6500c39425R702-R719

There seems to be code duplication across the test cases. Could these be made as "constants" and imported from another module?

Reply:
Moved to a new file `mock.data.js`.